onnxruntime/core/session/custom_ops.cc

```diff
@@ -865,6 +865,17 @@ KernelCreateInfo CreateKernelCreateInfo(const std::string& domain, const OrtCust
     def_builder.Provider(onnxruntime::kCpuExecutionProvider);
   }
 
+  if (op->version >= 18) {
+    int* input_index = nullptr;
+    int* output_index = nullptr;
+    size_t len = op->GetMayInplace(&input_index, &output_index);
+    if (len > 0) {
+      for (size_t i = 0; i < len; i++) def_builder.MayInplace(input_index[i], output_index[i]);
+      free(input_index);
```

Comment:
is input_index memory allocated in CustomOp library? Cross-DLL malloc/free may break the ABI compatibility.

Reply:
It is. Does the same issue happen when ORT-extension is using some ORT API like CreateMemoryInfo(), KernelContext_GetAllocator()? As the OrtMemoryInfo and OrtAllocator object are also allocated memory inside ORT

Reply:
Do I need to provide a function inside CustomOp library to do the release? if malloc and free happens in the same shared library, there won't be any issue, right?